apps/console/src/pages/navbar-pages/configure-metadata/ClaimedWebsiteStatus.svelte

```diff
@@ -37,7 +37,7 @@
     if (isValidUrl(value)) {
       http
         .post('api/claimed-website', {
-          url: value,
+          url: value.endsWith('/') ? value.slice(0, -1) : value,
```

Comment:
Unfortunately, the intention was to show validation error rather than silently change it for the user
> For the developer console, I don't think we should be silently trying to make corrections like this. What the user enters is what is used. If there's a problem, we should show errors, but the user should fix themselves.
[link](https://rdxworks.slack.com/archives/C03JJFEBR0V/p1706200077942549?thread_ts=1706192438.438799&cid=C03JJFEBR0V)

Reply:
also, this is not about checking it's status but rather what we put in the manifest and I don't see any changes about that?